src/lib/types/vec3-float32.js

```diff
@@ -0,0 +1,3 @@
+import r from 'restructure';
+
+export default new r.Array(r.floatle, 3);
```

Comment:
How would we differentiate between an array of three float values – currently `float3`, `vec3float32` – and an object with x, y, z properties – currently `Vec3Float`?